FindNearby/.gitignore

```diff
@@ -0,0 +1,40 @@
+# This file is used for Git repositories to specify intentionally untracked files that Git should ignore. 
+# If you are not using git, you can delete this file. For more information see: https://git-scm.com/docs/gitignore
+# For useful gitignore templates see: https://github.com/github/gitignore
+
+# Salesforce cache
+.sf/
+.sfdx/
+.localdevserver/
+deploy-options.json
+
+# LWC VSCode autocomplete
+**/lwc/jsconfig.json
```

Comment:
Please remove this line from `.gitignore` (it can stay in `.forceignore`), and add the following `jsconfig.json` file to `force-app/main/default/lwc/`:

```json
{
    "compilerOptions": {
        "experimentalDecorators": true
    }
}
```

This will get rid of TS-1219 errors in VSCode whenever we have annotations in our LWCs like `@api`. See [this PR comment](https://github.com/forcedotcom/LWC-Mobile-Samples/pull/10#discussion_r1069882021) for more discussion.